commit 04c912d14b1b227f4f11b80de0cac9ae4ca0a998 (refs/changes/95/995/1)
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2016-11-30T13:56:05-05:00 (2 years, 10 months ago)
    
    ./bin/triton fwrule update 172b1c32-2edd-4cfa-ac70-b2bcd0be0c5c description=hello

diff --git a/lib/do_fwrule/do_update.js b/lib/do_fwrule/do_update.js
index 8238c61..9bad3e4 100644
--- a/lib/do_fwrule/do_update.js
+++ b/lib/do_fwrule/do_update.js
@@ -37,7 +37,9 @@ function do_update(subcmd, opts, args, cb) {
 
     var id = args.shift();
 
-    vasync.pipeline({arg: {}, funcs: [
+    vasync.pipeline({arg: {cli: this.top}, funcs: [
+        common.cliSetupTritonApi,
+
         function gatherDataArgs(ctx, next) {
             if (opts.file) {
                 next();
